ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -531,8 +531,8 @@ presubmits:
       ci.openshift.io/generator: prowgen
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-installer-master-e2e-aws-upgrade
+    optional: true
```

Comment:
Seems `always_run` was not set

Reply:
`always_run: false` is the current state with no change--just needed to remove `skip_if_not_changed`. I think it's ok.

Reply:
ah, good catch